src/store/store.service.ts

```diff
@@ -65,4 +66,153 @@ export class StoreService {
       where,
     });
   }
+
+  async getStoreDeliveries(
+    partnerId: string,
+    skip?: number,
+    period?: 'today' | 'upcoming' | 'past',
+    search?: string,
+  ) {
+    const result = await this.prisma.order.findMany({
+      where: this.getDeliveryFilter(partnerId, period, search),
+      ...(skip && { skip }),
```

Comment:
@nedssoft There should be a `take` parameter that specifies the number of records to return, we can return 7 records at once. The take parameter works with the skip for the pagination to work fine